adafruit_fona/adafruit_fona.py

```diff
@@ -501,12 +499,11 @@ def send_sms(self, phone_number, message):
         return True
 
     def num_sms(self, sim_storage=True):
-        """Returns the number of SMS messages stored in memory, False if none stored.
+        """Number of SMS messages stored in memory, None otherwise.
```

Comment:
Should this be a property? I think you may have removed some returns from things that are functions.

Also, wouldn't it make sense to return 0 if no sms are stored? Maybe None if it's not configured to store them.

Reply:
@tannewt  
A property of the FONA object? 

I don't see any `returns` removed in https://github.com/adafruit/Adafruit_CircuitPython_FONA/pull/7/commits/b1b6a28d5c3609beb3c655047032e9ed1e9d22be. 

Returns 0 if no SMS stored on memory (internal _or_ SIM) in commit 96c996a

Reply:
By returns I meant the phrase "Returns..." in the comment. It's ok to have in the comment of a function.

Why are you returning None on an error? Seems like you should be raising an exception based on the response.

Reply:
> By returns I meant the phrase "Returns..." in the comment. It's ok to have in the comment of a function.

Ok, I put them back

> Why are you returning None on an error? Seems like you should be raising an exception based on the response.

Updated (3fa3b2b) PR to raise an exception that the [operating mode](https://www.developershome.com/sms/operatingMode2.asp) (text mode) was unable to be set. This (and returning a zero instead of `None` should work OK in `delete_all_sms`'s call to num_sms where it'll iterate through the sms memory .